lib/resources/dstu2/immunization.yaml

```diff
@@ -0,0 +1,79 @@
+---
+name: Immunization
+field_name_base_url: http://hl7.org/fhir/DSTU2/immunization-definitions.html#Immunization
+fields:
+
+- name: vaccineCode
+  required: 'Yes'
+  cardinality: 1..1
+  type: CodeableConcept
+  description: The vaccine that was administered or was to be administered.
+  example: |
+            {"vaccineCode": {
+              "coding": [
+                {"system": "http://www2a.cdc.gov/vaccines/iis/iisstandards/vaccines.asp?rpt=cvx",
+                 "code": "43"}],
+              "text": "Engerix-B (obsolete)"}}
+  binding:
+    description: Identifies the vaccine substance administered.
+    terminology:
+    - display: CVX
+      system: http://www2a.cdc.gov/vaccines/iis/iisstandards/vaccines.asp?rpt=cvx
+
+- name: site
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: The body site where the vaccine was administered.
+  example: |
+            {"site": {
+              "coding": [
+                {"system": "http://hl7.org/fhir/v3/ActSite",
+                 "code": "LT",
+                 "display": "left thigh"}],
+              "text": "left thigh"}}
+  binding:
+    description: The site at which the vaccine was administered.
+    terminology:
+    - display: v3 Code System ActSite
+      system: http://hl7.org/fhir/v3/ActSite
+
+- name: route
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: The path by which the vaccine product is taken into the body.
+  example: |
+            {"route": {
+              "text": "IV"}}
+  binding:
+    description: The path the administered medication takes to get into the body or into contact with the body.
+    terminology:
```

Comment:
We don't currently return any terminology system for this field - it's always freetext. Maybe list it as "currently not coded" ?